proto/executor/v1/executor.proto

```diff
@@ -42,6 +42,13 @@ message ProcessBatchResponse {
     uint32 cnt_binaries = 10;
     uint32 cnt_steps = 11;
     Error error = 12;
+    repeated ModifiedState modifications = 13;
```

Comment:
Is this `modifications` all the addresses that has been changed during a batch execution?
If it is that the case, I suggest to rename to `touchedAddresses` or `touchedState`

Reply:
Changed